jhipster-framework/src/main/java/tech/jhipster/web/util/HeaderUtil.java

```diff
@@ -15,22 +14,21 @@ public final class HeaderUtil {
 
     private static final Logger log = LoggerFactory.getLogger(HeaderUtil.class);
 
-    private HeaderUtil() {
-    }
+    private HeaderUtil() {}
 
     /**
      * <p>createAlert.</p>
      *
      * @param applicationName a {@link java.lang.String} object.
-     * @param message a {@link java.lang.String} object.
-     * @param param a {@link java.lang.String} object.
+     * @param alertMessage  a {@link java.lang.String} object.
+     * @param alertParam   a {@link java.lang.String} object.
      * @return a {@link org.springframework.http.HttpHeaders} object.
      */
-    public static HttpHeaders createAlert(String applicationName, String message, String param) {
+    public static HttpHeaders createAlertHeader(String applicationName, String alertMessage, String alertParam) {
```

Comment:
@DishaAnand You rename this method that is used in generator-jhipster, so pipelines fail